package/skelcd-control-openSUSE.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Jan 19 12:48:38 UTC 2022 - Ludwig Nussel <lnussel@suse.de>
+
+- Use NetworkManager always
```

Comment:
If I'm not wrong, a reference to Bugzilla/JIRA/Github entry is needed to please OBS.

Reply:
Not for Tumbleweed

Reply:
But is needed to please Jekins. See the result of the [public job](https://ci.opensuse.org/job/yast-skelcd-control-openSUSE-master/68/console)

> Stopping, missing new bugzilla or fate entry in the *.changes file.

Reply:
looks like you really like to make your own lives harder :) filed follow up pr